quipucords/quipucords/templates/registration/login.html

```diff
@@ -1,9 +1,28 @@
 {% extends "base.html" %}
 
+{% block includes %}
+  {% load static %}
```

Comment:
Optional: feels meta including a variable/param inside a block include, maybe move it outside